commons/src/main/scala/com/lynbrookrobotics/potassium/commons/drivetrain/Drive.scala

```diff
@@ -9,10 +9,18 @@ trait Drive {
   type DriveSignal
   type DriveVelocity
 
+  type DrivetrainHardware
```

Comment:
In the end, DrivetrainHardware and DrivetrainProps will be objects anyways, right?